jainr/azure-search-knowledge-mining
# intital bubble grouping interactive map

## Purpose
Applies UI change to interactive map enabling automatic bubble cluster of many crowded elements.

## Pull Request Type
What kind of change does this Pull Request introduce?

[ ] Bugfix
[X ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
Run the search UI. Enter search query *. On the interactive map zoom out until bubble clusters appear indicating size, color displays.

```

## What to Check
Verify that the following are valid
* bubble clustering appears delineated by color, size on zoom out.
